### PR TITLE
Homepage done

### PR DESCRIPTION
**Pull request by Charlotte-Ketteler**

### CODE REVIEW

File: components/Homepage/index.js
Comment (Kheirah) [inline components/Homepage/index.js:1]: You need to format this file with prettier.

File: components/Homepage/index.styled.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { styled } from "styled-components";
```
Comment (Kheirah) [inline components/Homepage/index.styled.js:1]:
```suggestion
import styled from "styled-components";
```

**Review by Kheirah — APPROVED** [review]
LGTM!

File: components/Homepage/index.js
Comment (Charlotte-Ketteler) [reply components/Homepage/index.js:1]: Sorry, I should have thought of that before! :)

**Pull request merged**